tests/gatewayapitests/graphlayer/httproute_test.go

```diff
@@ -1417,3 +1417,986 @@ func TestHTTPRouteParentFQDN(t *testing.T) {
 	akogatewayapitests.TeardownGateway(t, gatewayName1, DEFAULT_NAMESPACE)
 	akogatewayapitests.TeardownGatewayClass(t, gatewayClassName)
 }
+func TestHttpRouteWithValidAndInvalidGatewayListeners(t *testing.T) {
+	gatewayName := "gateway-hr-08"
```

Comment:
the naming should be gateway-hr-20 instead of 08

Reply:
Done